Source/Upstream: Yes, in v238-stable branch

From e439584ff90e8e1f47d2fc6f46e78ed8c8027feb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 9 Mar 2018 14:15:39 +0100
Subject: [PATCH 12/51] meson: use triple-quote delimition in one more place

(cherry picked from commit f83f8c70bd518e2b7a0011d82a2fb7074fb3a19a)
---
 src/journal-remote/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/journal-remote/meson.build b/src/journal-remote/meson.build
index 5fdc4cccd..cbe2be19c 100644
--- a/src/journal-remote/meson.build
+++ b/src/journal-remote/meson.build
@@ -61,6 +61,6 @@ if conf.get('ENABLE_REMOTE') == 1 and conf.get('HAVE_MICROHTTPD') == 1
         meson.add_install_script('sh', '-c',
                                  mkdir_p.format('/var/log/journal/remote'))
         meson.add_install_script('sh', '-c',
-                                 'chown 0:0 $DESTDIR/var/log/journal/remote &&
-                                 chmod 755 $DESTDIR/var/log/journal/remote || :')
+                                 '''chown 0:0 $DESTDIR/var/log/journal/remote &&
+                                    chmod 755 $DESTDIR/var/log/journal/remote || :''')
 endif
-- 
2.16.3

